data-reconciliation-app/docs/data-reconciliation.md

```diff
@@ -0,0 +1,69 @@
+# What is data reconciliation means in the context of the project?
+
+## _Reconciliation in finance_
```

Comment:
Per updated requirements, we will to make our app more generalized. I recommend we remove any specific LEI or banking language from this document. 

This top level section could say something along the lines of:
Generally speaking, data reconciliation is used describe a verification phase where the target data is compared against original source data to address a range of issues: 
- Missing records
- Missing values
- Incorrect values
- Duplicated records
- Badly formatted values
- Broken relationships across tables or systems

However, in many scenarios, there may not exist a single source of truth to compare your target data against. Rather, institutions can come together in a network to consolidate, compare and identify issues with their own data. One of the benefits of a CCF network is that these institutions can do so confidently, without actually sharing their data with one another. 

Some potential use cases may involve:
- In finance, a number of financial institutions coming together to predict the price of a derivative.
- In finance, a number of financial institutions coming together to reconcile reference data or ticker symbols. 
- In healthcare, a number of hospitals coming together to reconcile hospital records for patients.

Reply:
done